src/main/java/org/opensearch/geospatial/ip2geo/action/GetDatasourceRequest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.geospatial.ip2geo.action;
+
+import java.io.IOException;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j2;
+
+import org.opensearch.action.ActionRequest;
+import org.opensearch.action.ActionRequestValidationException;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+
+/**
+ * Ip2Geo datasource get request
+ */
+@Getter
+@Setter
+@Log4j2
```

Comment:
you are not using logger in this class too.

Reply:
Done